app/serializers/serializer_post_processing.rb

```diff
@@ -0,0 +1,18 @@
+# TODO: docs
+module SerializerPostProcessing
+  # TODO: docs
+  def serializable_object(options={})
+    super.tap {|hash|
+      modify_serializable_object(hash)
+    }
+  end
+
+  # TODO: docs
+  def modify_serializable_object(hash)
+    # TODO: having this method as an abstract overridable method is not very good,
```

Comment:
AMS upgrade is pain, because so many things have changed there. It is even possible that some features which are required for us are missing or impossible in a newer version. Upgrade `0.9` => `0.10` requires a lot of work. There is no migration guide, there is no proper documentation for `0.10`, and so many things are non-obvious. Moreover it is possible that there will be `1.0` release and it'll make some breaking changes again and will require another painful migration from `0.10` to `1.0` (see [Status of AMS](https://github.com/rails-api/active_model_serializers#status-of-ams)).

I tried to start the upgade, but I can see that it'll take a lot of time, so if we block this PR for AMS upgrade, it'll delay that for a month at least. I think that I can work on AMS upgrade at the meantime, but using 0.9 is totally okay for now -- there is no deprecation announced for it and it's not clear if it will happen anytime soon.